### DISCUSSION TITLE
Certain devices do not appear in home bridge from Smart Life app

### DISCUSSION THREAD

**Discussion by espinola17** [milo526/homebridge-tuya-web#399] [Q&A]
Example: I just added a new LED light strip. I have the same light strip in other rooms and successfully added via smart life and then after a restart of homebridge they were there. Well this new light strip will not show up. I’ve tried manually adding it by device ID from smart life app into homebridge TUYA web config but still can’t get it to show. I contacted smart life /TUYA and they told me I should check here. 

Any ideas ? I have some other devices like light switches that I can’t get to show in my homebridge either. But it’s only a small handful

**Comment by JCSEA2019** [comment]
I am also having trouble. I have one device in my smart life app but it does not show up in homebridge

**Reply by devicedev1** [reply]
I also have such a problem. The plugin successfully connects to the account, but does not see the device. Adding a virtual ID (app/about device) doesn't help. The devices are not visible.

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.

**Reply by espinola17** [reply]
I will post a reply by following your steps shortly.
